src/chat/Controller.java

```diff
@@ -0,0 +1,93 @@
+package chat;
+
+import javafx.application.Platform;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.input.KeyCombination;
+import javafx.scene.layout.BorderPane;
+import javafx.stage.Stage;
+
+import java.awt.*;
+import java.awt.datatransfer.*;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Controller {
+
+    @FXML
+    private MenuItem exitItem;
+    @FXML
+    private MenuItem copyItem;
+    @FXML
+    private MenuItem pasteItem;
+    @FXML
+    private TextArea messagesArea;
+    @FXML
+    private TextField newMessageArea;
+    private Stage stage;
+
+    private String userName = "JavaStudent";
+
+    public void setStage(Stage stage) {
+        this.stage = stage;
+        this.stage.setMinWidth(300);
+        this.stage.setMaxHeight(400);
+    }
+
+    @FXML
+    void initialize() {
+        exitItem.setAccelerator(KeyCombination.keyCombination("Ctrl+X"));
+        copyItem.setAccelerator(KeyCombination.keyCombination("Ctrl+C"));
+        pasteItem.setAccelerator(KeyCombination.keyCombination("Ctrl+V"));
+    }
+
+    @FXML
+    public void sendMessage(ActionEvent actionEvent) {
+        messagesArea.appendText(
+                String.format(
+                        "%s %s: %s\n",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.